services/port_manager/src/main/java/com/futurewei/alcor/portmanager/processor/FixedIpsProcessor.java

```diff
@@ -284,7 +284,7 @@ private void allocateRandomIpCallback(IRestRequest request) throws Exception {
             subnetIds.add(fixedIp.getSubnetId());
         }
 
-        getSubnetAndRoute(context, new ArrayList<>(subnetIds));
+        //getSubnetAndRoute(context, new ArrayList<>(subnetIds));
```

Comment:
@chenpiaoping Could you pls review this change? I raised a question on Slack channel regarding why we need to reevaluate subnet and route after allocating IP address.